src/Components/ADT3DSceneBuilder/ADT3DSceneBuilder.types.ts

```diff
@@ -153,6 +153,7 @@ export type BehaviorTwinAliasFormInfo = null | {
     twinAlias: IBehaviorTwinAliasItem;
     mode: TwinAliasFormMode;
     twinAliasIdx?: number;
```

Comment:
how can I get the search box to show up? when I edit a behavior I dont' get the option to search.
![image](https://user-images.githubusercontent.com/57726991/172911118-ff601222-9422-4ffc-af31-86371e24a0ea.png)

Reply:
good catch

Reply:
good catch, fixed